openlibrary/plugins/openlibrary/js/bulk-tagger/BulkTagger.js

```diff
@@ -516,14 +526,20 @@ export class BulkTagger {
         })
             .then(response => {
                 if (!response.ok) {
-                    new FadingToast('Batch subject update failed. Please try again in a few minutes.').show()
+                // Error handling
+                    this.isSubmitting = false;
+                    this.submitButton.disabled = false;
+                    this.submitButton.textContent = 'Submit';
+                    new FadingToast('Error submitting batch').show();
                 } else {
-                    this.hideTaggingMenu()
-                    new FadingToast('Subjects successfully updated.').show()
-
-                    // Update fetched subjects:
-                    this.updateFetchedSubjects()
-                    this.resetTaggingMenu()
+                // Success
+                    this.isSubmitting = false;
+                    this.submitButton.textContent = 'Submitted!';
+                    // Rest of success handling
+                    this.hideTaggingMenu();
+                    new FadingToast('Success!').show();
+                    this.updateFetchedSubjects();
+                    this.resetTaggingMenu();
```

Comment:
```suggestion
                    this.hideTaggingMenu();
                    new FadingToast('Subjects successfully updated.').show()
                    this.submitButton.textContent = 'Submit';
                    this.updateFetchedSubjects();
                    this.resetTaggingMenu();
```
- No need to change the order of operations here
- No need to change the toast message here
- Submit button's text should be reset to its original state.  Several submissions can occur without reloading the page.